README.md

```diff
@@ -9,13 +9,14 @@ to use cross platform tool to execute a set of kernels for example on compute cl
 OpenCL benchmarks or test and validate kernels. HDF5 files are used for the entire configuration and
 data handling. The list of kernels to be executed, all variables and the workgroup size is defined in
 the input HDF5 file. After execution, the output data, data copy- and runtime is written to the
-output HDF5 file.
+output HDF5 file. The GPU power consumption and temperature for supported Nvidia GPUs can also be logged automatically to the output HDF file.
 
 ## Usage
 To use toolkitCL the following needs to be installed:
 - OpenCL (headers and drivers)
 - HDF5
 - CMake
+- CUDA Toolkit (only for GPU power/temperature logging)
```

Comment:
Maybe a remark that CUDA will be detected automatically if present would be good? It doesn't need to be here but somewhere in the README would be good.